libs/sdk-core/src/breez_services.rs

```diff
@@ -907,6 +909,40 @@ impl BreezServices {
         Ok(self.btc_receive_swapper.refund_swap(req).await?)
     }
 
+    pub async fn initialize_liquid_wallet(&self) -> SdkResult<()> {
+        let mut wollet = self.liquid_wallet.lock().await;
+
+        *wollet = Some(
+            BreezWollet::new(WolletOptions {
+                signer: todo!(),
+                desc: todo!(),
+                db_root_dir: None,
+                electrum_url: None,
+                network: breez_sdk_liquid::Network::Liquid,
+            })?
+        );
+
+        Ok(())
+    }
+
+    pub async fn send_liquid_onchain(&self, amount_sat: u64) -> SdkResult<()> {
+        let mut lock = self.liquid_wallet.lock().await;
+
+        let wollet = lock
+            .as_mut()
+            .ok_or(SdkError::Generic { err: "Liquid wallet not initialized".to_string() })?;
+
+        let invoice = self
+            .node_api
+            .create_invoice(amount_sat * 1000, "liquid swap".to_string(), None, None, None, None)
+            .await?;
```

Comment:
Of course, this will need to be expanded. Ideally, we want to persist the invoice created (and probably even mark it as a swap), and poll for the funds to be received/use a webhook (like we currently do in `receive_onchain`)